dev/bin/interaction/create-action.sh

```diff
@@ -57,8 +57,8 @@ elif [ "${action_type}" = 'slack-plain' ]; then
    "action": {
       "type": "Slack",
       "parameters": {
-         "channel": "mc",
-         "webhookUrl": '"\"$SLACK_WEBHOOK_URL\""'
+         "slack_channel": "mc",
+         "slack_webhook_url": '"\"$SLACK_WEBHOOK_URL\""'
       }
```

Comment:
Just brainstorming: we could change the `type` to `Connector` and provide the connector type inside the `parameters`. In this way we can better separate what is a connector and what is not.